docs/csharp/reference-semantics-with-value-types.md

```diff
@@ -35,34 +35,34 @@ edit the *csproj* file and add the following node:
 
 You can use either "7.2" or "latest" for the value.
 
-## Specifying `in` parameters
+## Passing arguments by readonly reference
 
 C# 7.2 adds the `in` keyword to complement the existing `ref`
-and `out` keywords when you write a method that passes arguments
-by reference. The `in` keyword specifies that you are passing
+and `out` keywords to pass arguments
+by reference. The `in` keyword specifies passing
 the parameter by reference and the called method does not modify
-the value passed to it. 
+the value. 
 
 This addition provides a full vocabulary to express your design intent. 
-Value types are copied when passed to a called method when you do not
-specify any of the following modifiers. Each of these modifiers specify
+Value types are copied when passed to a called method when you don't
+specify any of the following modifiers. Each of these modifiers specifies
```

Comment:
perhaps this should be "specify any of the following modifiers in the method signature" (or called method), since they don't have to be specified at the call site.  Making that clear is especially important given the later discussion.